dask/dataframe/io/parquet.py

```diff
@@ -1107,51 +1056,62 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
     --------
     to_parquet
     """
-    is_ParquetFile = False
-    try:
-        import fastparquet
-        if isinstance(path, fastparquet.api.ParquetFile):
-            if path.open != fastparquet.util.default_open:
-                assert (re.match('.*://', path.fn)), \
-                       ("ParquetFile: Path must contain protocol" +
-                        " (e.g., s3://...) when using other than the default" +
-                        " LocalFileSystem. Path given: " + path.fn)
-
-            assert (engine in ['auto', 'fastparquet']), \
-                   ("'engine' should be set to 'auto' or 'fastparquet' " +
-                    'when reading from fastparquet.ParquetFile')
-            is_ParquetFile = True
-    except ImportError:
-            pass
-
-    if is_ParquetFile:
-        read = get_engine('fastparquet')['read']
-        if path.fn.endswith('_metadata'):
-            # remove '_metadata' from path
-            urlpath = path.fn[:-len('_metadata')]
-        else:
-            urlpath = path.fn
+    if isinstance(columns, str):
+        df = read_parquet(path, [columns], filters, categories, index,
+                          storage_options, engine, infer_divisions)
+        return df[columns]
```

Comment:
This becomes the way to make a series? 
The previous asserts on columns being a list could be dropped I think.

Reply:
Yes.  It seems simpler and centralizes a bunch of logic that was way further down.  This change is isolated to the first commit, which has many more deletions than additions.